packages/api/internal/handlers/sandbox_create.go

```diff
@@ -234,10 +233,7 @@ func (a *APIStore) PostSandboxes(c *gin.Context) {
 	telemetry.ReportEvent(ctx, "Created analytics event")
 
 	go func() {
-		err = a.db.UpdateEnvLastUsed(context.Background(), env.TemplateID)
-		if err != nil {
-			a.logger.Errorf("Error when updating last used for env: %s", err)
-		}
+		a.batcher.UpdateTemplateSpawnCount(env.TemplateID)
```

Comment:
We probably don't need to call this in goroutine now.

Reply:
I have a lock for sending the actual updates, so it could potentially slow it down